src/components/TextGenerationForm.tsx

```diff
@@ -149,7 +163,7 @@ function TextGenerationForm(): JSX.Element {
         </div>
 
         <div className="checkbox-container">
-          <label className="icon" htmlFor="linkedin">
+          <label tabIndex={0} className="icon" htmlFor="linkedin">
```

Comment:
I think the tabIndex should be in `input`, I tried it right now pressing Enter key does not check/uncheck the checkbox, can you take a closer look? thx